examples/cache-handler-redis/cache-handler-redis.js

```diff
@@ -1,39 +1,35 @@
 const { IncrementalCache } = require('@neshca/cache-handler')
-const { createHandler } = require('@neshca/cache-handler/redis-stack') // @neshca/cache-handler/redis-strings also available
+const createRedisCache = require('@neshca/cache-handler/redis-stack').default
+const createLruCache = require('@neshca/cache-handler/local-lru').default
 const { createClient } = require('redis')
 
-function createRedisClient(url) {
-  const client = createClient({
-    url,
-  })
-
-  client.on('error', (error) => {
-    console.error('Redis error:', error.message)
-  })
+const client = createClient({
+  url: process.env.REDIS_URL ?? 'redis://localhost:6379',
+})
 
-  return client
-}
+client.on('error', (error) => {
+  console.error('Redis error:', error.message)
+})
 
-async function connect(client) {
-  try {
-    await client.connect()
-  } catch (error) {
-    console.error('Redis connection error:', error.message)
-  }
-}
+IncrementalCache.onCreation(async () => {
+  // read more about TTL limitations https://caching-tools.github.io/next-shared-cache/configuration/ttl
+  const useTtl = false
 
-const client = createRedisClient(
-  process.env.REDIS_URL ?? 'redis://localhost:6379'
-)
+  await client.connect()
 
-connect(client).then(() => {
-  console.log('Redis connected')
-})
-
-IncrementalCache.onCreation(
-  createHandler({
+  const redisCache = await createRedisCache({
     client,
+    useTtl,
+  })
+
+  const localCache = createLruCache({
```

Comment:
Hi @ezeparziale! If Redis fails, you can use it as a fallback instead of a slow file system. But if you use a shared file system setup, you should turn it off. I added a [new section](https://caching-tools.github.io/next-shared-cache/configuration/use-file-system#the-usage-of-local-lru-handler-instad-of-file-system-cache) to the docs.

Reply:
Thanks for prompt reply!

I think this the best approach to run in k8s with some replicas.

```javascript
IncrementalCache.onCreation(async () => {
    let redisCache, localCache;

    const useTtl = true;

    if (process.env.REDIS_AVAILABLE) {
        await client.connect();
 
        redisCache = await createRedisCache({
            client,
            useTtl,
        });
     
        localCache = createLruCache({
            useTtl,
        });
    }

    const cache = [redisCache, localCache];
    
    return {
        cache,
        useFileSystem: false
    };
});
```